al2.pkr.hcl

```diff
@@ -59,30 +65,25 @@ build {
     ]
   }
 
-  provisioner "file" {
-    source      = "files/29-ecs-banner-begin.sh.amzn2"
```

Comment:
Q: Just curious, if we're removing these sections for `29-ecs-banner-begin` and `31-ecs-banner-finish` do we still need them to be defined on L.4-5? If we do, where will these files come from?

Reply:
>If we do, where will these files come from? 
Never mind, I see we;re essentially doing the same thing on L68-74 and L.77-86!